diff --git a/target/linux/generic/kernel-6.12 b/target/linux/generic/kernel-6.12
index 73c2f33a10..ee47400ffc 100644
--- a/target/linux/generic/kernel-6.12
+++ b/target/linux/generic/kernel-6.12
@@ -1,2 +1,2 @@
-LINUX_VERSION-6.12 = .63
-LINUX_KERNEL_HASH-6.12.63 = 9502c5ffe4b894383c97abfccf74430a84732f04ee476b9c0d87635b29df7db3
+LINUX_VERSION-6.12 = .64
+LINUX_KERNEL_HASH-6.12.64 = 2696e06a6c1beb29f880fac01ca18ac2fdd89fa9604621806293c2efcbfc36ce
