diff --git a/target/linux/generic/kernel-6.12 b/target/linux/generic/kernel-6.12
index db25695394..641b59483e 100644
--- a/target/linux/generic/kernel-6.12
+++ b/target/linux/generic/kernel-6.12
@@ -1,2 +1,2 @@
-LINUX_VERSION-6.12 = .49
-LINUX_KERNEL_HASH-6.12.49 = 234621e146dacce2241049555d550e4f7a6bde67ccd7ef232d47ac8145425526
+LINUX_VERSION-6.12 = .50
+LINUX_KERNEL_HASH-6.12.50 = c435bd74d1c21fc5a950781a50d78bae2b93944144694843359948ad3afc72a5
