diff --git a/target/linux/generic/kernel-6.12 b/target/linux/generic/kernel-6.12
index 73c2f33a10..ee47400ffc 100644
--- a/target/linux/generic/kernel-6.12
+++ b/target/linux/generic/kernel-6.12
@@ -1,2 +1,2 @@
-LINUX_VERSION-6.12 = .61
-LINUX_KERNEL_HASH-6.12.61 = 1a69745105528676f12f29dc2494945d96cb23666dcc5223794abc22415f1735
+LINUX_VERSION-6.12 = .62
+LINUX_KERNEL_HASH-6.12.62 = d17449476378181cbb650d6d08b3b55e16dfe2a82a6047516c4f644f8d8aaba7
