diff --git a/target/linux/generic/kernel-6.12 b/target/linux/generic/kernel-6.12
index c786e6ed44..d638b2f7bf 100644
--- a/target/linux/generic/kernel-6.12
+++ b/target/linux/generic/kernel-6.12
@@ -1,2 +1,2 @@
-LINUX_VERSION-6.12 = .67
-LINUX_KERNEL_HASH-6.12.67 = 16805dc62e1fa5ef8a3f466f3f44a2efb171b5206d6840ced4ba5475cf12c432
+LINUX_VERSION-6.12 = .68
+LINUX_KERNEL_HASH-6.12.68 = d367c7504bd4da520dd01eb08125d2d0ac088bc8af4cd56d236f2074dd4225b7
